### DISCUSSION TITLE
Settings or mods to enable death at end of map

### DISCUSSION THREAD

**Discussion by WinterIsPretty** [ppy/osu#16692] [Ideas]
I do know that you guys are trying to make an HP universal within all 4 modes but unless you guys are going with Taiko HP, this is applicable. These are all my ideas for this concept.

1. Since SD and PF have to do with dying under certain conditions, there could be a setting for that group for if you died within the map you died at the end and not during the map.

2. If not a mod, maybe a setting that you enable that will kill you at the end of the map instead of during the map.

3. I think another idea is to do placebo mods where it would display that you are playing with certain mods but they actually don't do anything. And it wouldn't save to the replay file or anything since they do nothing to the actual gameplay. When my friends or I play the game with NF, some of us would tend to perform way better than nomod even though all it does is not kill you.

placebo mods may replace the mods that you have enabled already. if you had HR on and you turned on no mods it would hide the mods you had enabled or replace it with EZ or anything.

Anyways I really think one of these would increase the quality of the game because there is so much psychic stuff that affects your gameplay. I know, for example, if someone plays on private servers they tend to make a lot of better scores than if you were on bancho. Many people who play taiko say they wish other modes had similar HP where you would die at the end and not during the map because you can play the map whether you die or not without the NF mod so that would also appeal to all of those players.

**Comment by uzervlad** [comment]
I feel like it defeats the purpose of SD and PF mods, so having a separate mod incompatible with those two would probably be a better option

**Comment by frenzibyte** [comment]
Seems quite similar to https://github.com/ppy/osu/discussions/13068. The part about "placebo mods" for NF specifically and what not sounds like what has been proposed in https://github.com/ppy/osu/issues/9844.

**Comment by smoogipoo** [comment]
We do plan on having Taiko use the same HP system as the other rulesets.

**Comment by WinterIsPretty** [comment]
If taiko will have the same HP as other modes, it would probably be a good idea to have a setting that will enable death at the end of the map instead of during the map
